packages/eas-cli/src/metadata/context.ts

```diff
@@ -52,8 +53,8 @@ export async function createMetadataContextAsync(params: {
     params.profileName
   );
 
-  const exp = params.exp;
-  const user = params.credentialsCtx.user;
+  const exp = params.exp ?? getExpoConfig(params.projectDir);
+  const user = params.credentialsCtx ? params.credentialsCtx.user : undefined;
```

Comment:
I think this is the same:
```suggestion
  const user = params.credentialsCtx?.user;
```